OracleWebLogic/samples/1221-multihost/create-container.sh

```diff
@@ -23,6 +24,8 @@ fi
 eval "$(docker-machine env $swarm $machine)"
 
 docker run -d $DOCKER_CONTAINER_INSTANCE_OPTIONS \
+  -e MS_HOST=$name\
```

Comment:
@mriccell, did you find any issue happening because these vars were not set? If yes, can you please provide some details? Just to get an understanding...